src/components/ReportDetails.vue

```diff
@@ -19,8 +19,134 @@ export default {
   name: 'ReportDetails',
   components: { DomainListReports, DomainListDoughnuts },
   computed: {
+    /**
+     * @return {string}
+     */
     getReportId () {
       return this.report.id.toString()
+    },
+    /**
+     * @return {{}}
+     */
+    getReport () {
+      let urls = this.report.reports
+      let url = this.getUrl(urls, this.report.domain)
+      let data = {}
+
+      // Set main report
+      if (urls[url]) {
+        Reflect.set(data, 'report', urls[url].report)
+      }
+
+      // Set main scanner codes
+      this.fetchReport(urls[url].report, item => {
+        if (!Reflect.get(data, item.scanner_code)) {
+          Reflect.set(data, item.scanner_code, [])
+        }
+      })
+
+      // Fetch URL keys
+      for (let reportUrl in urls) {
+        // Skip main domain as it gets print out anyway
+        if (reportUrl === url) {
+          continue
+        }
+
+        // Special handling for mail servers as those use other scanners
+        if (reportUrl.includes('mx')) {
```

Comment:
Siehe Chat: hier darf die prüfung nicht sein, ob mx im domainnamen auftaucht sondern ob beim zugehörigen Report einer der Mail-spezfiischen Scanner (imap, pop, smtp) auftaucht. Hintergrund: dass "mx" in der Domain auftaucht ist eher die Ausnahme als die Regel

Reply:
Jup. Habs sowieso noch mal umgebaut. Sollte so hinhauen.